test/emcc_O2_hello_world.fromasm.clamp

```diff
@@ -8976,9 +8973,7 @@
     (global.set $__THREW__
      (local.get $0)
     )
-    (global.set $threwValue
-     (local.get $1)
-    )
+    (nop)
```

Comment:
Should we be running another vacuum after this to clean up the nops?

Reply:
Do you mean adding another pass after "simplify-globals" in addDefaultGlobalOptimizationPostPasses?  or calling something in the SimplyGlobals pass itself?

Reply:
Yeah adding another pass after `simplify-globals`.